src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2317,22 +2317,15 @@ Uninstall the following packages: {0}?</value>
   <data name="PreferencesViewVisualSettingsTab" xml:space="preserve">
     <value>Visual Settings</value>
   </data>
-  <data name="PreferencesWindowLanguages" xml:space="preserve">
```

Comment:
@pinzart90 seems removing this broke something - I think these need to be copied to the other resex instead of deleted.

Reply:
I don;t remember changing this...I'll try to revert